### PR TITLE
Sc 42 dashboard filtering

### PR DESCRIPTION
**Pull request by jfho**

### CODE REVIEW

**Review by Spobendonis — COMMENTED** [review]
There should be some form of documentation regarding the use of prime-react (why is it necessary and why have you chosen prime-react in particular)

**Review by Spobendonis — COMMENTED** [review]
The navbar (health graphs) button currently takes us to href=healthgraphs instead of href=health_graphs. This is arguably a little out of scope, but this is probably the best story to fix it.

**Comment by AstralEik** [comment]
It would be good test design to rework the one giant FE test into smaller FE tests. Basically a test for each // there currently exists.
After that it looks good :3

**Review by AstralEik — DISMISSED** [review]
very pog, hope we in the future can have a blank option on the filters :333

**Review by Spobendonis — DISMISSED** [review]
Filtering systemet fejler med responsive design lige nu, som den er ulæselig på små skærme

**Review by Spobendonis — COMMENTED** [review]
The latest version of this branch does not seem to consistently work with serial number selection

**Comment by Cromski** [comment]
I would change the default value of "clickedMenu" to true. I'd rather have to click the burger menu to filter, than to land on the page on the phone, and not being able to see any graphs to start. 
Other than that, i love it

**Comment by AstralEik** [comment]
I cannot currently

**Review by Spobendonis — COMMENTED** [review]
Serial number select seems to still not work on my device. It seems as though the values present to select are out of date and don't exist in the context of the program.

File: cypress/e2e/homepage.cy.js
Comment (Spobendonis) [inline cypress/e2e/homepage.cy.js:1]: I think the Iframe used should be health-graphs, as overview-of-devices-copy is not very clear

**Review by AstralEik — APPROVED** [review]

**Review by AstralEik — APPROVED** [review]

**Pull request merged**